src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,23 +1,50 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Status } from '../../enum/Status';
+
+export const TodoFilter: React.FC = ({
+  onStatus,
+  onQuery,
+}) => {
+  const [status, setStatus] = useState(Status.All);
+  const [query, setQuery] = useState('');
+  const options = Object.entries(Status).map(([key, value]) => (
+    <option value={value}>{key}</option>
+  ));
+
+  const handleStatus = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const selecteed = event.target.value as Status;
+
+    setStatus(selecteed);
+    onStatus(selecteed);
+  };
+
+  const handleQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
![image](https://github.com/mate-academy/react_redux-list-of-todos/assets/55200530/21dd1027-6902-4183-8748-6eec9a8a369d)